src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
This works...and is how I debugged the prefetch runtime.  The debugging doesn't change the behavior of the routine and is not compiled in unless the preprocessor macro is defined.  The reason for the complexity is that I also needed to limit how much debug came out (e.g. I just wanted to see the first X prefetch ops in detail...and any out of bounds errors that happen anytime else).  Without the limits, if too much debug came out through log_printf, then logcat would crash.  I can remove all of this now as well.